pkg/webhooks/mutator/hyperConvergedMutator_test.go

```diff
@@ -264,9 +265,17 @@ var _ = Describe("test HyperConverged mutator", func() {
 
 			AfterEach(func() {
 				util.GetClusterInfo = getClusterInfo
+
+				nodeList := &corev1.NodeList{}
+				err := cli.List(context.TODO(), nodeList, client.MatchingLabels{"node-role.kubernetes.io/worker": ""})
+				Expect(err).ToNot(HaveOccurred())
+				for i := range nodeList.Items {
+					node := &nodeList.Items[i]
+					Expect(cli.Delete(context.TODO(), node)).To(Succeed())
+				}
 			})
 
-			DescribeTable("check EvictionStrategy default", func(SNO bool, strategy *kubevirtcorev1.EvictionStrategy, patches []jsonpatch.JsonPatchOperation) {
+			DescribeTable("check EvictionStrategy default", func(SNO bool, strategy *kubevirtcorev1.EvictionStrategy, patches []jsonpatch.JsonPatchOperation, workerNodeArches []string) {
```

Comment:
This test is a unit test. Adding and removing nodes using the client is overkill. We can just create a new fake client with the required nodes for the specific entry.

e.g.
```suggestion
			DescribeTable("check EvictionStrategy default", func(SNO bool, strategy *kubevirtcorev1.EvictionStrategy, patches []jsonpatch.JsonPatchOperation, workerNodeArches []string) {
			
				cr.Status.InfrastructureHighlyAvailable = ptr.To(!SNO) // <<<=== this change is not related to this PR, but please apply it

				cr.Spec.EvictionStrategy = strategy
				cli = commontestutils.InitClient(createFakeNodes(workerNodeArches))
				mutator = initHCMutator(s, cli)
```

Reply:
done